package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1055,7 +1059,7 @@ def get_attr(attrname, default):
             vals['occupancy'] = occupancies[i]
             vals['tempFactor'] = tempfactors[i]
             vals['segID'] = segids[i][:4]
-            vals['element'] = guess_atom_element(atomnames[i].strip())[:2]
+            vals['element'] = elements[i][:2]
```

Comment:
Should this have an `upper()` call? I've seen PDB files with both mixed case or just all upper case. We enforce UpperLower within MDA, but it would be best to write out in the manner that's consistent, thoughts?

Reply:
Yeah I like `.capitalize()`. That being said, given that we already validate and capitalize elements on input, if the user reeeeeeally wants bespoke capitalisation then maybe we should let them.

Reply:
So we normalize all the inputs to be `capitalize()`, which is where this gets complicated, I prefer then converting back to `all caps/upper()` elements because I think we see these a bit more often in PDBs, but if users had a specific preference then we'd be overriding that :/

Reply:
It seems that [all-caps used to be the spec](https://lists.sdsc.edu/pipermail/pdb-l/2004-January/001397.html), and here is [an example of capitalized symbols causing issues](https://gitlab.com/ase/ase/-/issues/78) in NGLView -- I vote for upper case unless/until someone specifically raises an issue about it.

Reply:
That's convincing enough for me, I've changed it to upper in bd220dc :)